let s = getInput()
throw new Error("ERROR")
returnOutput(s)
